README.md

```diff
@@ -186,6 +190,123 @@ func CallerWorkflow(ctx workflow.Context) error {
 }
 ```
 
+## Options
+
+### Service
+
+#### (nexustemporal.v1.service).disabled
```

Comment:
Wondering why you need two bools. I'm thinking we only need an `enabled` flag and treat the absence of this option as unspecified. WDYT?